src/content/refi/index.md

```diff
@@ -0,0 +1,67 @@
+
+Carbon credits and carbon markets existed long before the term ReFi was coined. The seeds were planted at the Earth Summit in Rio de Janeiro in 1992, when 160 countries agreed to the [UN Framework Convention on Climate Change (UNFCCC)](https://unfccc.int/). There are now several government-mandated carbon markets (‘compliance markets’) that aim to establish a carbon price via laws or regulations controlling the supply of permits to be distributed by national, regional, and global regimes. The VCM is an entirely separate market in which private actors voluntarily buy and sell carbon credits that represent certified removals or reductions of greenhouse gasses in the atmosphere. However, despite its development over recent decades, the VCM continues to suffer from a variety of issues, such as highly fragmented liquidity, opaque transaction mechanisms, high fees, very slow trading speed, and lack of scalability to meet the necessary 10–20x growth that the [Taskforce on Scaling the Voluntary Carbon Markets](https://www.iif.com/tsvcm) predicts is needed in the coming decade for the VCM to deliver on its potential to mitigate the climate crisis.
+
+Transitioning the VCM to the new blockchain-based **digital carbon market (DCM)** might be an opportunity to upgrade the existing technology for validating, transacting and consuming carbon credits. Blockchains allow for publicly verifiable data, access for a broad range of users and more liquidity.
+
+ReFi projects employ blockchain technology to alleviate many of the problems of the traditional market:
+
+- In the DCM, liquidity is concentrated in a small number of [liquidity pools](https://www.gemini.com/cryptopedia/what-is-a-liquidity-pool-crypto-market-liquidity) that can be freely traded by anyone. Large organizations as well as individual users can use these pools without manual searches for sellers/buyers, participation fees, or prior registration.
+- All transactions are recorded on public blockchains. The path each carbon credit takes due to trading activity is traceable forever as soon as it is made available in the DCM.
+- Transaction speed is nearly instant. Securing large amounts of carbon credits via the legacy markets can take days or weeks, but this can be achieved in a few seconds in the DCM.
+- Trading activity occurs without intermediaries, which charge high fees. Digital carbon credits represent a [62% cost improvement compared to equivalent traditional credits](https://www.klimadao.finance/blog/klimadao-analysis-of-the-base-carbon-tonne), according to data from one analytics firm.
+- The DCM is scalable and can meet the demands of individuals and multinational corporations alike.
+
+Many of the issues responsible for slow growth and impracticality in the VCM, as well as its inability to scale, are either mitigated or solved completely by the transition to a fast, energy-efficient DCM and the applications built on it.

```

Comment:
```suggestion
-**Liquidity is concentrated in a small number of liquidity pools** that can be freely traded by anyone. Large organizations as well as individual users can use these pools without manual searches for sellers/buyers, participation fees, or prior registration.
- **All transactions are recorded on public blockchains**. The path each carbon credit takes due to trading activity is traceable forever as soon as it is made available in the DCM.
- **Transaction speed is nearly instant**. Securing large amounts of carbon credits via the legacy markets can take days or weeks, but this can be achieved in a few seconds in the DCM.
- **Trading activity occurs without intermediaries**, which charge high fees. Digital carbon credits represent a [62% cost improvement compared to equivalent traditional credits](https://www.klimadao.finance/blog/klimadao-analysis-of-the-base-carbon-tonne), according to data from one analytics firm.
- **The DCM is scalable** and can meet the demands of individuals and multinational corporations alike.
```